R/deployApp.R

```diff
@@ -50,9 +50,10 @@
 #'   on second and subsequent deploys, the title will be unchanged.
 #' @param envVars A character vector giving the names of environment variables
 #'   whose values should be synchronised with the server (currently supported by
-#'   Connect only). The values of the environment variables are sent over an
-#'   encrypted connection and are not stored in the bundle, making this a safe
-#'   way to send private data to Connect.
+#'   Connect only). The values of the environment variables should be set in the
+#'   current session with [Sys.setenv()] or via an `.Renviron` file. Values are
```

Comment:
I don't think it's good practice to recommend `Sys.setenv()` since that'll result in the value of the env var appearing in your `.Rhistory`. It might be good to say that explicitly here.

Reply:
Good point. I took another pass and rewrote this sentence:

> A character vector giving the names of environment variables whose values should be synchronised with the server (currently supported by Connect only). The values of sensitive environment variables should be set in the current session via an `.Renviron` file or with the help of a credential store like [keyring](https://r-lib.github.io/keyring/index.html). Values are sent over an encrypted connection and are not stored in the bundle, making this a safe way to send private data to Connect.